releasenotes/notes/0.8-prelude-3057999fac96bf43.yaml

```
@@ -0,0 +1,7 @@
+---
+prelude: >
+    From this release, Qiskit Machine Learning requires Qiskit `1.0` or above, with important
+    changes and upgrades, such as the introduction of Quantum Bayesian inference and  the migration
+    of a subset of Qiskit Algorithms features to Qiskit Machine Learning. These changes are
+    part of the process to build full compatibility with the version-2 Qiskit primitives, to be
+    introduced in version `0.8` of Qiskit Machine Learning in Q4 of 2024.
```

```suggestion
    introduced in version `0.9` of Qiskit Machine Learning in Q4 of 2024.
```
Given this will be 0.8 this is now the next version right. ` in Q4 of 2024` do you want to say `currently planned for Q4 2024` so it does not seem quite so definitive and gives room to change if needed